src/pages/ListTableList/_mock.ts

```diff
@@ -0,0 +1,146 @@
+import { Request, Response } from 'express';
+import { parse } from 'url';
+import { TableListItem, TableListParams } from './data.d';
+
+// mock tableListDataSource
+let tableListDataSource: TableListItem[] = [];
+
+for (let i = 0; i < 10; i += 1) {
+  tableListDataSource.push({
+    key: i,
+    disabled: i % 6 === 0,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 0](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4665582)